analysis/PE_AMI cox model code stata.do

```diff
@@ -0,0 +1,91 @@
+* Script to run PE and ATE outcomes in the electively unvaccinated population in STATA
+* Electively unvaccinated is a whole population sample
+
+clear
```

Comment:
Remove this line. Clear is not necessary as every action starts in a fresh Stata docker container.